package.json

```diff
@@ -11,7 +11,7 @@
     "author": "Nordic Semiconductor ASA",
     "license": "SEE LICENSE IN LICENSE",
     "engines": {
-        "nrfconnect": ">=4.4.1"
+        "nrfconnect": ">=5.0.0"
```

Comment:
@datenreisender please advice if we should update engine in this case (as we've updated electron). Looks as a correct thin, but not exactly required in this case

Reply:
It should still run with launcher 4.4.1, right? I see no changes, which require 5.0, so I see no reason to bump this requirement.

That shared depends on a newer version of Electron now should also be fine. Only if we have any code which uses newer features on Electron (here or in shared) this would become relevant.